lib/engine/game/g_1849/step/bond.rb

```diff
@@ -30,7 +30,12 @@ def description
           end
 
           def log_skip(entity)
-            super if @game.bonds? && @game.issue_bonds_enabled
+            print "hi #{can_take_loan?(entity)}"
```

Comment:
A stray debugging statement has made it through to the pull request.
```suggestion
```

Reply:
removed